bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/BundleProcessor.java

```diff
@@ -214,6 +235,38 @@ private CSVMetadataHolder createMetadataHolder(AnnotationInstance csvMetadata, C
             maintainers = mh.maintainers;
         }
 
+        final var linksField = csvMetadata.value("links");
+        CSVMetadataHolder.Link[] links;
+        if (linksField != null) {
+            final var linkAnn = linksField.asNestedArray();
+            links = new CSVMetadataHolder.Link[linkAnn.length];
+            for (int i = 0; i < linkAnn.length; i++) {
+                links[i] = new CSVMetadataHolder.Link(
+                        ConfigurationUtils.annotationValueOrDefault(linkAnn[i], "name",
+                                AnnotationValue::asString, () -> null),
+                        ConfigurationUtils.annotationValueOrDefault(linkAnn[i], "url",
+                                AnnotationValue::asString, () -> null));
+            }
+        } else {
+            links = mh.links;
+        }
+
+        final var iconField = csvMetadata.value("icon");
```

Comment:
The icon is already dealt with automatically if users add an appropriately named file in `src/main/resources`. See https://github.com/quarkiverse/quarkus-operator-sdk/blob/main/bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/builders/CsvManifestsBuilder.java#L168-L180. Can we find a way to unify this with explicit metadata?

Reply:
Perhaps we could provide an alternative options to user?
1. Select file from `src/main/resources`
<img width="265" alt="Screen Shot 2022-10-21 at 9 28 41 AM" src="https://user-images.githubusercontent.com/1379213/197208058-abce4ee3-c9ac-4793-aab2-7185cbaea8ea.png">

2. Use base64 string
<img width="304" alt="Screen Shot 2022-10-21 at 9 29 26 AM" src="https://user-images.githubusercontent.com/1379213/197208083-5af7b2d8-10c4-4156-af9f-359dbf070172.png">